features/admin_center_dashboard.feature

```diff
@@ -229,6 +229,7 @@ Feature: Dashboard
     Then the "Discover and add the RSS feeds and social media accounts listed on the following page:" field should contain "https://www.nih.gov"
 
     When I follow "Display"
+    And the use redesigned results page is disabled for the site "agencygov"
```

Comment:
Rather than checking that this behavior hasn't changed (if `use_redesigned_results_page` is false that you can still see this one sidebar link), we should be confirming that the sidebar links have actually disappeared for users that *are* opted in to the redesign ( `use_redesigned_results_page=true`).

There's already a setup block in place for this scenario here that can be used for these confirmations: https://github.com/GSA/search-gov/blob/main/features/admin_center_manage_display.feature#L655